### ISSUE TITLE
[LANGUAGE] Ruby

### ISSUE THREAD

**Issue by dylburger** [PipedreamHQ/pipedream#2160]

**Comment by konung** [comment]
I was poking around docs & repos, and see that at some point in the early alpha, Ruby was the target language for the Pipedream SDK? It looks like the project changed the architecture since then . A lot 😄 

I was also wondering if there is a plan to support Ruby (crystal, nim, zig, would be too niche I think 😮 ) as a language for triggers & actions, alongside python, bash, js and go?

**Comment by dylburger** [comment]
@konung yes, we've since deprecated the SDK but since we use Ruby / Rails internally, it was something we were excited to offer! Not many people ended up using the Ruby client, but to your second question — yes, we'd still like to ship Ruby as a first-class language in Pipedream workflows, and we'll update this issue when we have updates. Ideally, we'd provide a mechanism to bring your own runtime so that you could hack in any language. That is the vision.

**Comment by dylburger** [comment]
This is the repo where we track all issues. We don't explicitly prioritize issues publicly, so you won't see an ordered backlog, but this is where you should raise new issues / follow existing ones.

However, our current roadmap _roughly_ follows [requests in GitHub, ordered by total reactions](https://github.com/PipedreamHQ/pipedream/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-desc). We're either working on most of the stuff in the top 10, or are planning to in 2023.
